api/tests/opentrons/cli/test_cli.py

```diff
@@ -24,13 +26,21 @@ def _list_fixtures(version: int) -> List[Path]:
 
 
 @pytest.mark.parametrize("defpath", _list_fixtures(5))
-def test_analyze(defpath: Path) -> None:
-    """Should return with no errors and a non-empty output."""
+def test_analyze(defpath: Path, tmp_path: Path) -> None:
+    """Should return with no errors and a none empty output."""
```

Comment:
```suggestion
    """Should return with no errors and a non empty output."""
```

Looks like this typo fix got accidentally reverted in a merge.